gui/src/app/state/transactions.rs

```diff
@@ -87,7 +87,11 @@ impl State for TransactionsPanel {
                 Err(e) => self.warning = Some(e),
                 Ok(txs) => {
                     self.warning = None;
-                    for tx in txs {
+                    // FIXME: This is a quick hack to ensure order is consistent in order to quickly
+                    // check and switch between transactions.
+                    let mut txs_sorted = txs;
+                    txs_sorted.sort_by_key(|tx| tx.tx.txid());
+                    for tx in txs_sorted {
```

Comment:
Seems to be worth on its own?

Reply:
There might be a better way to order the transactions, and possibly other changes that would be helpful when viewing the transactions list, so shall we leave this for a separate PR?

I think the only GUI change required for RBF, apart from the negative fees issue for missing coins that will be addressed in #820 and #825, is to leave the replacement PSBT status as "Pending" instead of "Deprecated" so that it can be signed and broadcast. I've made that change, but still need to check if we can set the replaced PSBT to "Deprecated" once the replacement has been confirmed.

Reply:
Sure.

Reply:
PSBTs with input coins that have been spent by another confirmed transaction will be shown as Deprecated.

I've made a further change so that PSBTs with only inputs that no longer exist are shown as Deprecated. This can happen if the PSBT was spending the outputs of a transaction that was replaced or otherwise no longer exists.